### DISCUSSION TITLE
SVB Form are WPF.

### DISCUSSION THREAD

**Discussion by zs-3** [VBAndCs/sVB-Small-Visual-Basic#87] [General]
I want to know are SVB Forms WPF as i am creating my extension for small basic (ZS).
because i decompiled SVB Library and found the xaml and style part is not possible in Winform.

**Comment by VBAndCs** [comment]
Small Basic and Small Visual Basic are created with WPF.
And you don't need to decompile sVB as the source code is fully published in this repo.
The sVB Forms are more like WinForms but they are written with WPF.

________________________________
From: ZS ***@***.***>
Sent: Monday, September 23, 2024 7:28 AM
To: VBAndCs/sVB-Small-Visual-Basic ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [VBAndCs/sVB-Small-Visual-Basic] SVB Form are WPF. (Discussion #87)


I want to know are SVB Forms WPF as i am creating my extension for small basic (ZS).
because i decompiled SVB Library and found the xaml and style part is not possible in Winform.
